modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageManagerImpl.java

```diff
@@ -1116,7 +1116,7 @@ private void onSafeTimeAdvanced(HybridTimestamp time) {
     /**
```

Comment:
I guess we also call resetPeers in MetaStorageManagerImpl, we either should implement retry logic here or create a  todo. Or which might be slightly better idea, move retry logic to the raft level, like we do in common sendWithRetry in order not to duplicate retry code.

Reply:
I don't think it makes any sense now when MS groups do not have sequence token - bare retry will make things worse.